### PR TITLE
Allows `render_destroy` action to be triggered from Shortcake Bakery.

### LINKED ISSUE #219: shortcode-ui.render_destroy from Shortcake plugin is not being fired.
`insertAction` method for WP Post media frame is being overridden.

And because of that the default event `shortcode-ui.render_destroy` from Core Shortcake plugin is not being fired.

Ref: 

https://github.com/wp-shortcake/shortcake/commit/f7364a30f1ef4b1b8fe848fbade866950d21c349#diff-e896f34e64a0875262f930016b24aee8R126

https://github.com/wp-shortcake/shortcake-bakery/commit/a77b4062d12fa5dd676f0476a54b4a89dcaa9ef7#diff-ab3cd71f9dd659dda2e8239ccabe433eR180

### PR DESCRIPTION
**Pull request by desaiuditd**
Fixes #219.

`insertAction` method was wrongly being overridden. Because of this, the `shortcode-ui.render_destroy` action was not getting fired. The action was added in the Shortcake UI plugin after this code was added. Hence.

### CODE REVIEW

File: assets/js/build/shortcake-bakery-add-embed-media-frame.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+(function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
```
[the remark this answers is not in the document]
Comment (desaiuditd) [reply assets/js/build/shortcake-bakery-add-embed-media-frame.js]: I'm not sure.

I might be missing something to build the assets properly.

All I did was to run `grunt` default task in the repo.

**Comment by aprea** [comment]
LGTM. I've confirmed that the `shortcode-ui.render_destroy` event is firing when inserting new / updating existing shortcodes.

File: assets/js/build/shortcake-bakery-add-embed-media-frame.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (goldenapples) [reply assets/js/build/shortcake-bakery-add-embed-media-frame.js]: This repo doesn't have a package-lock or anything of the sort. I'd assume that the differences here are just caused by an updated browserify version. I'm getting the same build output on a fresh npm install, and the built scripts work fine, so `¯\_(ツ)_/¯` I think its good.

**Comment by goldenapples** [comment]
Unrelated build failures aside, this looks good and makes sense. There's really no reason to be overriding the insertAction method from the Shortcake frame here. Thanks for the PR!

**Pull request merged**